launch/faulkner.launch

```diff
@@ -0,0 +1,49 @@
+<launch>
```

Comment:
Why is this file called "faulkner.launch"? Should we have a more specific name?

Reply:
we can create, this is just to differentiate simulation vs device launch.